src/internal_events/kubernetes_logs.rs

```diff
@@ -18,6 +19,14 @@ impl InternalEvent for KubernetesLogsEventReceived<'_> {
 
     fn emit_metrics(&self) {
         counter!("processed_events_total", 1);
+        if let Some(name) = self.pod_name {
+            counter!(
+                "events_in_total", 1,
+                "pod_name" => name.to_owned(),
+            );
+        } else {
+            counter!("events_in_total", 1);
+        }
```

Comment:
I recommend `match` for this:
```suggestion
        match self.pod_man {
            Some(name) => 
                counter!("events_in_total", 1, "pod_name" => name.to_owned()),
            None => counter!("events_in_total", 1),
        }
```